docs/pages/access-controls/idps/saml-attribute-mapping.mdx

```diff
@@ -0,0 +1,198 @@
+---
+title: SAML IdP Attribute Mapping
+description: How to map user attributes to custom SAML response
+h1: SAML Attribute Mapping
+---
+
+### Prerequisites
+
+(!docs/pages/includes/no-oss-prereqs-tabs.mdx!)
+
+- (!docs/pages/includes/tctl.mdx!)
+- User with permission to create Service Provider resource. The preset `editor` role has this permission.
```

Comment:
We introduce the Service Provider resource in one subsection of the SAML reference, which I'm not sure we can expect all readers of this page to have read. We should briefly explain this resource somewhere on this page.

Reply:
Hmmm, so the feature "Attribute Mapping" is not a standalone feature but an extension of core SAML app configuration functionality. I mean user can surely bump straight into attribute mapping docs. But in practice, we can fairly expect users to have gone though our initial SAML guide, without which, attribute mapping itself is not possible.